cli/src/build/onboarding/android/ui/app.tsx

```
@@ -29,8 +29,9 @@ import { requestBuildInternal } from '../../../request.js'
 import type { CiSecretEntry, CiSecretSetupAdvice, CiSecretTarget } from '../../ci-secrets.js'
```

ESLint

[error] Expected "../../ci-secrets.js" (type-parent) to come before "../../../request.js" (value-parent).

(perfectionist/sort-imports)